tests/unit/Commands/DiffControllerTest.php

```
@@ -23,9 +27,229 @@ final class DiffControllerTest extends TestCase
+    {
+        $this->seedStubAndDestination('config/params.php', "content\n");
+        $this->writeLockEntry('config/params.php');
+
+        $currentPath = "{$this->tempDir}/config/params.php";
+
+        MockerState::addCondition(
+            'yii\\scaffold\\Commands',
+            'file_get_contents',
+            [$currentPath, false, null, 0, null],
+            false,
+        );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Xepozz\InternalMocker\MockerState' in method 'testActionIndexReturnsErrorWhenCurrentFileReadFails'. (undefined)

(StaticAccess)